src/tests/Rockets.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { useSelector } from 'react-redux';
+import Rockets from '../pages/Rockets/Rockets';
+
+// Mocking react-redux's useSelector
+jest.mock('react-redux');
+
+// Mocking axios
+jest.mock('axios', () => ({
+  get: jest.fn(),
+}));
+
+test('Rockets component renders correctly', () => {
+  // Mock the useSelector behavior
+  useSelector.mockReturnValue({
+    rockets: [
+      {
+        id: 1,
+        name: 'Falcon 9',
+        description: 'A powerful rocket developed by SpaceX.',
+        flickr_images: ['image1.jpg', 'image2.jpg'],
+        reserved: false,
+      },
+      // Add more mock rockets as needed
+    ],
+  });
+
+  const { container } = render(<Rockets />);
+
+  expect(container).toMatchSnapshot();
+});
```

Comment:
- [ ] Great work implementing some required changes from the previous reviewer. However, a test is still failing at this moment. Wrapping all your test cases inside `describe` block would solve the issue of the failing test I guess as suggested by the previous reviewer. I got the test passed when I wrapped your test case inside `describe` block. Kindly take a look at the both case scenario.

### Before
![failed-test](https://github.com/pushkar1995/space-travelers-hub/assets/26597432/3e4d9423-7ba6-4634-b963-2fa01271e3cf)

### After

```
   
// Mocking react-redux's useSelector
jest.mock('react-redux');

// Mocking axios
jest.mock('axios', () => ({
  get: jest.fn(),
}));

describe('Renders components', () => {
  it('Rockets component renders correctly', () => {
    // Mock the useSelector behavior
    useSelector.mockReturnValue({
      rockets: [
        {
          id: 1,
          name: 'Falcon 9',
          description: 'A powerful rocket developed by SpaceX.',
          flickr_images: ['image1.jpg', 'image2.jpg'],
          reserved: false,
        },
        // Add more mock rockets as needed
      ],
    });

    const { container } = render(<Rockets />);

    expect(container).toMatchSnapshot();
  });
})

```
![passed-test](https://github.com/pushkar1995/space-travelers-hub/assets/26597432/bd279419-2fce-430a-bd31-07126792cc4a)